Testing/Test-Validation.ps1

```diff
@@ -25,7 +25,7 @@ param(
     [Parameter(Mandatory)][string]$ResourceGroup,
     [Parameter()][string]$WebAppName = $ResourceGroup,
     [Parameter()][string]$Organization = 'protacon',
-    [Parameter()][string]$Repository = 'repository-validator'
+    [Parameter()][string]$Repository = 'repository-validator-testing'
```

Comment:
This check is also done as a part of CI pipeline (test & warmup) and we are not guaranteed that this repository-validator-testing repository exists, so this shouldn't be touched.